### ISSUE TITLE
Incorrect parsing of hex numbers containing `0b`

### ISSUE THREAD

**Issue by Jarcho** [attilathedud/vscode-data-converter#11]
This is caused by the regex replacement before parsing. Should probably be `\b0b` assuming the word boundary is supported.

**Comment by attilathedud** [comment]
Good catch! I probably won't have time to implement it in the near term, but will put it on the back burner. Feel free to submit a PR and I'll get it merged if you want it done sooner.
